.github/workflows/glpi.yml

```
@@ -13,7 +13,6 @@ on:
         default: ""
       php-version:
         type: string
```

As for the `image-suffix` attribute.

```suggestion
        required: false
        type: string
```